test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/MultipleProcessesIntegrationTest.java

```diff
@@ -20,6 +22,7 @@
 import org.kie.server.client.UserTaskServicesClient;
 import org.kie.server.controller.client.KieServerControllerClient;
 
+@Category(Smoke.class)
```

Comment:
This can be marked as JBPMonly too.

Reply:
Yes, it can. But this test run scenario with database and there isn't configuration for drools and database.
So do we want to skip this scenario for drools (skip test in test results) OR not include in drools test run (test is not in test results)?

Reply:
Good point, I overlooked that. It is ok in current state.